packages/rest/src/__tests__/integration/rest.application.integration.ts

```diff
@@ -163,6 +166,73 @@ describe('RestApplication (integration)', () => {
     await client.get(response.header.location).expect(200, 'Hi');
   });
 
+  context('mounting an express router on a LoopBack application', async () => {
```

Comment:
Please add a test to verify the order of precedence of external vs. static-asset routes.

```ts
app.staticAssets('/test', /* path to some dir */);
app.mountExpressRouter('/test', /* a router */);
// ...

await client.get(
 // a URL matching a static asset and an Express route too
).expect(
  // the express route was invoked
);
```